Mathlib/Topology/Algebra/Valued/NormedValued.lean

```diff
@@ -126,4 +139,22 @@ def toNormedField : NormedField L :=
         exact ⟨fun a b hab => lt_of_lt_of_le hab (min_le_left _ _), fun a b hab =>
             lt_of_lt_of_le hab (min_le_right _ _)⟩ }
 
+section NormedField
+
+/-- When a field is valued, one inherits a `NormedField`. Local instance to avoid
+a typeclass loop or non-defeq topology or norms. -/
+local instance : NormedField L := Valued.toNormedField L Γ₀
```

Comment:
If it does not cause problems with inference loops, could you also add the reverse `NormedField.toValued` local instance inside the `NormedField` namespace?

Reply:
done in [fce8539](https://github.com/leanprover-community/mathlib4/pull/17561/commits/fce8539cd02f471cd3ff85db12aff7eabad75fc0)